trunk/SUAVE/Methods/Missions/Segments/Climb/Linear_Speed_Constant_Rate.py

```diff
@@ -40,7 +39,7 @@ def initialize_conditions(segment,state):
     v_x   = np.sqrt( v_mag**2 - v_z**2 )
     
     # pack conditions    
-    conditions.frames.inertial.velocity_vector[:,0] = v_x
+    conditions.frames.inertial.velocity_vector[:,0] = v_x[:,0]
```

Comment:
Yes. v_x is a vector now. It was a scalar before... v_z is a scalar because it is constant rate. There was an error here prior because it was copy of a CSCR segment, which has v_x and v_z as both scalars.